source/core.py

```diff
@@ -61,6 +61,8 @@ def doStartupDialogs():
 		gui.messageBox(_("Your gesture map file contains errors.\n"
 				"More details about the errors can be found in the log file."),
 			_("gesture map File Error"), wx.OK|wx.ICON_EXCLAMATION)
+	if not config.conf['update']['askedAllowUsageStats']:
+		gui.AskAllowUsageStatsDialog.run()

```

Comment:
This should probably only ask if updating is turned on? The check box is hidden in that case already right? 

In fact, this should probably not be shown unless updating is possible. For instance, when run from the launcher or portable copies.